src/component/contents/exploration/__tests__/DraftEditorContents.react-test.js

```diff
@@ -0,0 +1,142 @@
+/**
```

Comment:
We have a file called `DraftEditorContent.react-test.js` however since I plan to move the logic for rendering tree outside of that file it made sense to create a separate test file as opposed to add to the existing one.